configure

```diff
@@ -21,7 +21,9 @@ use strict;
 use warnings;
 
 ## Version of OCI's TAO to download
-my $tao_version = '2.2a';
+my $oci_tao_version = '2.2a';
+## Version of DOC Group ACE/TAO to download, uses the ACE version number
+my $doc_tao_version = '6.4.3';
```

Comment:
Yes, it should be changed to my idea, else if someone has for example OpenDDS 3.12 and runs configure two years in the future he will get the latest ACE/TAO from that moment, that is probably not tested and will not work